element-android/pipeline.yml

```
@@ -39,6 +39,7 @@ steps:
     commands:
       - "./gradlew clean lintGplayRelease assembleGplayDebug --stacktrace"
       - "mv vector/build/outputs/apk/gplay/debug/*.apk ."
+    key: "elementApp"
```

we should surely not have two steps with the same key?

```suggestion
    key: "gplay_debug"
```